src/Management/src/Endpoint/Services/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Steeltoe.Common;
+using Steeltoe.Management.Endpoint.Middleware;
+using Steeltoe.Management.Endpoint.Refresh;
+
+namespace Steeltoe.Management.Endpoint.Services;
+
+/// <summary>
+/// Add services used by the Refresh actuator.
```

Comment:
This is just draft PR, so we can focus on the design and usability of the Services and important details and we can clean all these other things up - otherwise its hard to focus on feedback because of the noise.